man/subscription-manager.8

```diff
@@ -76,9 +76,6 @@ must be passed as system arguments in a non-interactive session.
 .IP
 2. unregister
 
-.IP
-3. attach
-
 .IP
 4. auto-attach
```

Comment:
The list is now missing `3.`

Reply:
Yeah, I know, but there will be lot of removal. I would like to fix this during final changes of man page.